projects/packages/publicize/src/class-connections.php

```diff
@@ -22,20 +22,24 @@ class Connections {
 	 * Get all connections.
 	 *
 	 * @param array $args Arguments
-	 *                - 'clear_cache': bool Whether to clear the cache.
-	 *                - 'test_connections': bool Whether to run connection tests.
+	 *                - 'ignore_cache': bool Whether to ignore the cache and fetch the connections from the API.
 	 * @return array
 	 */
 	public static function get_all( $args = array() ) {
 
-		$run_tests = $args['test_connections'] ?? false;
-
 		$is_wpcom = ( new Host() )->is_wpcom_simple();
 
 		if ( $is_wpcom ) {
-			$connections = Connections_Controller::get_connections( $run_tests );
+			$connections = Connections_Controller::get_connections();
```

Comment:
I know it wasn't introduced here, but it feels like we're coupling things by having this library class dependent on a REST controller.

Perhaps we should look at the code organisation in a follow-up?

Reply:
Yeah, that change needs to be a PR on its own.